enableEpochs.toml

```diff
@@ -285,50 +285,32 @@
+    CurrentRandomnessOnSortingEnableEpoch = 2
 
     # StakeLimitsEnableEpoch represents the epoch when stake limits on validators are enabled
     # Should have the same value as StakingV4Step1EnableEpoch that triggers the automatic unstake operations for the queue nodes
-    StakeLimitsEnableEpoch = 13
+    StakeLimitsEnableEpoch = 2
 
     # StakingV4Step1EnableEpoch represents the epoch when staking v4 is initialized. This is the epoch in which
     # all nodes from staking queue are moved in the auction list
-    StakingV4Step1EnableEpoch = 13
+    StakingV4Step1EnableEpoch = 2
 
     # StakingV4Step2EnableEpoch represents the epoch when staking v4 is enabled. Should have a greater value than StakingV4Step1EnableEpoch.
     # From this epoch, all shuffled out nodes are moved to auction nodes. No auction nodes selection is done yet.
-    StakingV4Step2EnableEpoch = 14
+    StakingV4Step2EnableEpoch = 3
 
     # StakingV4Step3EnableEpoch represents the epoch in which selected nodes from auction will be distributed to waiting list
-    StakingV4Step3EnableEpoch = 15
+    StakingV4Step3EnableEpoch = 4
 
     # AlwaysMergeContextsInEEIEnableEpoch represents the epoch in which the EEI will always merge the contexts
-    AlwaysMergeContextsInEEIEnableEpoch = 13
+    AlwaysMergeContextsInEEIEnableEpoch = 2
 
     # CleanupAuctionOnLowWaitingListEnableEpoch represents the epoch when duplicated data cleanup from auction list is enabled in the condition of a low waiting list
     # Should have the same value as StakingV4Step1EnableEpoch if the low waiting list has not happened, otherwise should have a greater value
-    CleanupAuctionOnLowWaitingListEnableEpoch = 13
+    CleanupAuctionOnLowWaitingListEnableEpoch = 2
 
-    # DynamicESDTEnableEpoch represents the epoch when dynamic NFT feature is enabled
-    DynamicESDTEnableEpoch = 895
-
-    # EGLDInMultiTransferEnableEpoch represents the epoch when EGLD in multitransfer is enabled
-    EGLDInMultiTransferEnableEpoch = 895
-
-    # CryptoOpcodesV2EnableEpoch represents the epoch when BLSMultiSig, Secp256r1 and other opcodes are enabled
-    CryptoOpcodesV2EnableEpoch = 895
-
-    # UnjailCleanupEnableEpoch represents the epoch when the cleanup of the unjailed nodes is enabled
-    UnJailCleanupEnableEpoch = 895
-
-    # RelayedTransactionsV3EnableEpoch represents the epoch when the relayed transactions V3 will be enabled
-    RelayedTransactionsV3EnableEpoch = 895
-
-    # FixRelayedBaseCostEnableEpoch represents the epoch when the fix for relayed base cost will be enabled
-    FixRelayedBaseCostEnableEpoch = 895
-
-    # MultiESDTNFTTransferAndExecuteByUserEnableEpoch represents the epoch when enshrined sovereign cross chain opcodes are enabled
-    MultiESDTNFTTransferAndExecuteByUserEnableEpoch = 9999999
+    # UseGasBoundedShouldFailExecutionEnableEpoch represents the epoch when use bounded gas function should fail execution in case of error
+    UseGasBoundedShouldFailExecutionEnableEpoch = 1
```

Comment:
why 1? Perhaps 2 or 3 to align with previous declared flags

Reply:
update to 4, highest one